packages/components/src/composite/stories/index.story.tsx

```diff
@@ -11,35 +11,31 @@ import { isRTL } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import {
-	Composite,
-	CompositeGroup,
-	CompositeRow,
-	CompositeItem,
-	useCompositeStore,
-} from '..';
+import * as Composite from '..';
 import { UseCompositeStorePlaceholder, transform } from './utils';
 
 const meta: Meta< typeof UseCompositeStorePlaceholder > = {
-	title: 'Components/Composite (V2)',
+	title: 'Components/Composite',
 	component: UseCompositeStorePlaceholder,
 	subcomponents: {
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		Composite,
+		Root: Composite.Root,
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		CompositeGroup,
+		Group: Composite.Group,
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		CompositeRow,
+		Row: Composite.Row,
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		CompositeItem,
+		Item: Composite.Item,
 	},
-	tags: [ 'status-private' ],
 	parameters: {
```

Comment:
I think there's something messed up with these, the default story doesn't have any controls:

![Screenshot 2024-07-18 at 16 29 28](https://github.com/user-attachments/assets/8dfa7578-49a7-430b-a8d9-5391c40065e2)

Or if that's intentional, it can definitely use some love and display all available props as controls.

I think that's pre-existing, because I see this on `trunk`:

![Screenshot 2024-07-18 at 16 31 39](https://github.com/user-attachments/assets/1d2e8f28-7611-46cd-9620-be9c23a4a9dd)

Reply:
That's indeed a decision that was taken by whoever worked on the component before. I'll see if it's an easy change to apply or if there were larger implications.

Reply:
Totally fine to be fixed in another PR since it's pre-existing behavior.

Reply:
I actually ended up manually adding type information to Storybook in this PR.